### PR TITLE
add my darkmode

### PR DESCRIPTION
**Pull request by MarceleSilv**
Description:
It's a switch that works in a fun way, aiming to reduce light exposure while reading and help with eye strain.

-Please add this contribution pull-requests to Hacktoberfest2022. 

https://codepen.io/marcelesilv/pen/yLjrEzz

### CODE REVIEW

File: README-zh_CN.md [hunk 1]
```diff
@@ -288,6 +288,15 @@
 ***
 &nbsp;
 
+## <a id="DarkMode"></a>Dark Mode - Day and nigth
```
Comment (cht8687) [inline README-zh_CN.md:291]: this section should be part of Chinese doc. 

you can either drop this change or use Google translator to translate the texts

**Comment by cht8687** [comment]
please resolve the conflicts so that we can merge this example.

**Comment by MarceleSilv** [comment]
Hello, I would love to work on this subject again, please.

**Comment by aayush105** [comment]
> Hello, I would love to work on this subject again, please.

Ok you can create a PR related to it, and read CONTRIBUTING.md file for the steps @MarceleSilv

**Pull request closed without merging**